Library/Homebrew/rubocops/keg_only.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require "rubocops/extend/formula"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      class Keg < FormulaCop
+        def audit_formula(node, _class_node, _parent_class_node, _body_node)
+          return unless find_node_method_by_name(node, :keg_only?)
```

Comment:
Facing some difficulty here. Not sure if this is the right way to access the `keg_only?` method. I've tried using the `find_every_func_call_by_name` and `find_node_method_by_name` methods as well but that doesn't seem to work as well. Since I could not find any documentation regarding these methods, I would appreciate pointers to proceed with this step. Thank you!

Reply:
That would be correct but it should in this case be `keg_only` without the `keg_only?` because it looks like this in formulae:
```
keg_only ...
```